src/langcheck/augment/en/_jailbreak_template.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from .._common._jailbreak_template import _jailbreak_template
+
+AVAILABLE_JAILBREAK_TEMPLATES = [
```

Comment:
I think a comment above each of these options would be helpful that includes:
- a quick description of the jailbreak
- link to the template file so the user can see the full prompt template easily

Reply:
Done.